analysis/import/GitLogParser/README.md

```diff
@@ -32,35 +32,66 @@ The names of authors are saved when the --add-author flag is set.
 
 ## Usage
 
-### Creating the repository log for metric generation
+### Creating required files on the fly | repo-scan
```

Comment:
Let's name this a bit different.
What about "Scan a local git repository on your machine". By that it should be more clear to the user what he can expect by the following descriptions